python/jupyter_leaflet/src/controls/GeomanDrawControl.ts

```diff
@@ -0,0 +1,528 @@
+import { unpack_models, WidgetView } from '@jupyter-widgets/base';
+import { GeoJSON } from 'leaflet';
+import L from '../leaflet';
+import { LayerShapes } from '../definitions/leaflet-extend';
+import { LeafletControlModel, LeafletControlView } from './Control';
+import 'leaflet/dist/leaflet.css';
+import '@geoman-io/leaflet-geoman-free';
+import '@geoman-io/leaflet-geoman-free/dist/leaflet-geoman.css';
+
+export class LeafletGeomanDrawControlModel extends LeafletControlModel {
+  defaults() {
+    return {
+      ...super.defaults(),
+      _view_name: 'LeafletGeomanDrawControlView',
+      _model_name: 'LeafletGeomanDrawControlModel',
+      current_mode: null,
+      hide_controls: false,
+      data: [],
+      marker: {},
+      circleMarker: { markerStyle: {} },
```

Comment:
```suggestion
      circlemarker: { markerStyle: {} },
```
should match trait name